app/javascript/topNavigation/utilities.js

```diff
@@ -145,11 +145,11 @@ export async function getInstantClick(waitTime = 2000) {
  */
 export function initializeMobileMenu(menuTriggers, moreMenus) {
   menuTriggers.forEach((trigger) => {
-    trigger.addEventListener('click', toggleBurgerMenu);
+    trigger.onclick = toggleBurgerMenu;
```

Comment:
Refactor (potentially blocking): @Zhao-Andy, curious why this was converted from a more standard `addEventListener`.

Reply:
👋 Suzanne caught an issue earlier, and I left a comment that's resolved now about it: https://github.com/forem/forem/pull/12612#discussion_r579309338

> So this resolves the issue you caught @aitchiss, which happened because InstantClick would add a second click event on top of the original. So the toggle would go open -> close instantly after the second page navigation.

Reply:
Ah, understood. There might be a better way to handle this, but for now, I think this is fine. I would add a comment in the code though explaining this.